README.md

```diff
@@ -77,6 +77,7 @@ We will read recent papers from top venues to understand trends in GenAI systems
 |         | **Fine Tuning / Alignment**
 | Feb  8  | [LoRA: Low-Rank Adaptation of Large Language Models](https://openreview.net/forum?id=nZeVKeeFYf9) (Required)
 |         | [Training Language Models to Follow Instructions with Human Feedback](https://proceedings.neurips.cc/paper_files/paper/2022/hash/b1efde53be364a73914f58805a001731-Abstract-Conference.html) (Required)
+|         | [S-LoRA: Serving Thousands of Concurrent LoRA Adapters](https://arxiv.org/abs/2311.03285)
```

Comment:
Move this under inference Feb 20.

Reply:
I feel like topics in week of Feb 8 is too ML, and this one is quite related to fine tuning and focuses more on the system side

Reply:
Hmm. True.